apps/web/src/pages/CommunityInterests/sections/FindANewCommunity.tsx

```diff
@@ -0,0 +1,283 @@
+                      "I’m not looking for work in this community right now.",
+                    id: "S+GQc9",
+                    description:
+                      "Label for the community not interested in work opportunities radio option",
+                  }),
+                },
+              ]}
+              rules={{
+                required: intl.formatMessage(errorMessages.required),
+              }}
+              disabled={formDisabled}
+            />
+            <RadioGroup
+              idPrefix="trainingInterest"
+              name="trainingInterest"
+              legend={intl.formatMessage({
+                defaultMessage: "Interest in training opportunities",
+                id: "H1UEd5",
+                description:
+                  "Label for the input for choosing interest level in a training opportunity",
+              })}
+              items={[
+                {
+                  value: "true",
+                  label: intl.formatMessage({
+                    defaultMessage:
+                      "I’m interested in being considered for training or development opportunities within this community.",
+                    id: "oEYCLO",
+                    description:
+                      "Label for the community interest in training opportunities radio option",
+                  }),
+                  contentBelow: intl.formatMessage({
+                    defaultMessage:
+                      "By indicating you’re interested in training opportunities, you agree to share your profile information with recruiters and hiring managers within this functional community.",
+                    id: "3TEZds",
+                    description:
+                      "Context for the 'I'm interested in training opportunities within this community' radio option",
+                  }),
+                },
+                {
+                  value: "false",
+                  label: intl.formatMessage({
+                    defaultMessage:
+                      "I’m not interested in training opportunities right now.",
+                    id: "cLBubr",
+                    description:
+                      "Label for the community not interested in training opportunities radio option",
+                  }),
+                },
+              ]}
+              rules={{
+                required: intl.formatMessage(errorMessages.required),
+              }}
+              disabled={formDisabled}
+            />
+            {/* work stream section */}
+            {workStreamOptions.length ? (
+              <div
+                data-h2-display="base(flex)"
+                data-h2-flex-direction="base(column)"
+                data-h2-gap="base(x0.5)"
+              >
+                <span>
+                  {intl.formatMessage({
+                    defaultMessage:
+                      "Please select any of the work streams described below that you would consider working within.",
```

Comment:
None of the work streams are described, they are only the title of the work stream. Also "below" is directional language that should be avoided for accessibility purposes.
```suggestion
                      "Please select any of the work streams listed that you would consider working within.",
```

Reply:
Good catch, in Figma I had changed this to "Please select any of the work streams within which you'd like to work." Perhaps this came through too late, or wasn't updated everywhere. Will double check.